RELEASE_NOTES.md

```diff
@@ -36,33 +38,33 @@ accompanying blog post.
 
 - New Sigmoid display transform module.
 
-- Two new highlight recovery algorithm are proposed: in-paint opposed
+- Two new highlight recovery algorithms are proposed: in-paint opposed
   and segmentation. The in-paint opposed has proved to be very stable
-  and good in a lot of pictures so it has been made the default now.
+  and good in a lot of pictures, so it has been made the default now.
 
 - A full rework of the image display has been done. An initial work
-  has been to create an easy to use image display on the central
-  area. This routine supports all display mode: focus-peaking and
+  has been to create an easy-to-use image display on the central
+  area. This routine supports both focus-peaking and
   color assessment. This routine is now used for the darkroom area,
-  for the second window and the duplicates.
+  for the second window, and the duplicates.
 
-  So the second window gets the focus-peaking display and more
+  So, the second window gets the focus-peaking display and more
   importantly the color assessment mode which is an important feature.
 
-  To achieve that properly a new routine has been created which run a
+  To achieve that properly, a new routine has been created which runs a
   separate pixelpipe which is fully equivalent to the darkroom
```

Comment:
'which' occurs twice in a sentence. Perhaps
..., a new routine has been **created. It runs a separate pixelpipe, which** is...

Reply:
Don't worry about this for now. It needs a rewrite and it's a pain to quibble about language in github comments. I'll try to make it sound better.